ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -60,7 +60,13 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
 
     // Associate the connection details with the session
     let session;
-    const { session_id, address, port, credentials } = connectionDetails;
+    const {
+      session_id,
+      address,
+      port,
+      credentials,
+      expiration: expiration_time,
```

Comment:
The `connectionDetails` are returned from the CLI ,desktop client leverages CLI to perform a connection, [here the code](https://github.com/hashicorp/boundary-ui/blob/main/ui/desktop/electron-app/src/models/session.js#L54-L66). 
Therefore, if you look at `connectionDetails` you are not receiving `expiration_time`, you are receiving `expiration`. You can test this also connecting to a target using CLI.


That said, What are we actually trying here? we are defining `expiration: expiration_time` at destructuring time but then assigning `expiration_time: expiration`? 
- This is redundant, you could have done `expiration_time` at destructuring and assigning time.
- That is making the linter to not pass.
- As described above, the assignments are incorrect.